### DISCUSSION TITLE
Primeiro emprego

### DISCUSSION THREAD

**Discussion by Kaedh** [frontendbr/forum#2084] [Dicas]
Pessoal
Sempre programei como um hobby, porém de um meses pra cá comecei a focar em usar esse hobby como minha profissão. Desde então entrei em um curso de análise  e desenvolvimento, e diariamente estudo e pratico em projetos pessoais usando React/Redux. Estou agora procurando meu primeiro emprego/estágio na área porem me sinto um pouco perdido em como fazer isso de forma assertiva. gostaria de saber de vocês o que fariam hoje se estivessem e busca do primeiro emprego na área .

**Comment by ninetails** [comment]
Bom, o que eu faria ao buscar o primeiro emprego _tendo a bagagem de conhecimento* que tenho hoje_...

<sup>\* sobre contratação, não sobre conhecimento técnico</sup>

São dicas que eu falo pra todo mundo que vejo que tá começando... Antes de tudo, _conecte-se_ e não seja invisível. Deixe teu LinkedIn impecável, teu GitHub... E então participe de eventos de TI. Puxe conversa, adicione pessoas... Veja o que as empresas estão utilizando, faça amizades, agrege contatos... Amplie teu _Networking_.

Dado isso, aplique-se a vagas. No começo vc ainda vai se sentir meio perdido mas a cada entrevista você vai passar tanto por RH quanto por possivelmente etapas técnicas conversando com outros desenvolvedores. Caso chegar nesta etapa, você vai ter respostas superficiais do que as empresas estão utilizando para saber o que está em tendência ou o que é legal de se estudar, caso tiver como deficiência em sua formação profissional. Além disso, você também vai ficando mais desinibido para outros processos seletivos. Também vai pegando _trends_ de entrevistas, perguntas frequentemente lançadas etc... Claro, lembrando sempre de manter-se ético e vc vai pegando o _feeling_ do que as empresas estão procurando.

...

Depois junte esses dois 😃 Networking e conhecimento em contratações.

---

Só outro ponto que eu diria pra dar atenção é que tente aumentar suas chances também... Pois entre candidatos, aquele com diferencial terá mais chances de ser contratado.

Mas é papo de coaching né: não dá pra saber dos outros candidatos numa mesma seleção, mas dá pra perseverar enquanto continua estudando até que uma hora as chances fiquem favoráveis pra ti e você se torne o candidato com mais diferenciais dentre a seleção.

**Reply by Kaedh** [reply]
Era esse o tipo de dica que eu estava buscando, confesso que essa parte de não ser invisível na internet me deixa um pouco perdido ainda mas estou pensando em coisas que posso fazer pra melhorar isso, inclusive ser mais participativo em fóruns de TI, como aqui e alguns que sigo no reddit ( se souber de formas de se expor mais, agradeço ).
Muito obrigado pelas dicas, espero que daqui a alguns meses eu possa me chamar de programador também kkk
